configure.ac

```diff
@@ -355,6 +355,10 @@ AC_ARG_ENABLE(native,
 
 AC_ARG_ENABLE(snmp-dest, [ --enable-snmp-dest Enable snmp destination (default: auto)],, enable_snmp_dest="auto")
 
+AC_ARG_ENABLE(config-database,
+	      [  --enable-config-database            Enable building config-database. (default: no)]
```

Comment:
Please add this option to the configure line in dbld/tarball, and add the pip dependencies to the default docker image.